extend.md

```diff
@@ -54,7 +54,14 @@ load("@bazel_gazelle//:def.bzl", "DEFAULT_LANGUAGES", "gazelle_binary")
 
 gazelle_binary(
     name = "gazelle",
-    languages = DEFAULT_LANGUAGES,
+    languages = [
+        "@rules_python//gazelle", #use gazelle from rules_python
+        "@bazel_gazelle//language/go", #built in rule from gazelle for golang
+        "@bazel_gazelle//language/proto", #built in rule from gazelle for protos
+        # must be after the proto extension (order matters)
+        "@build_stack_rules_proto//language/protobuf", #use gazelle to create protobufs see https://github.com/stackb/rules_proto for more  info
```

Comment:
I would rather not include an external plugin in the documentation for fear that it will look like an endorsement by the project. I'm sure that stack.build's rules_proto is great, but I've not used it and so I'd rather we make up another 3rd party plugin.

Could we go with something like

```
        "@external_repository//language/gazelle"  # External languages can also be added here. 
```

Reply:
I completely understand, I went ahead and implemented your solution since it solves both a generic plugin while still showing an usable example.